config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -26,10 +26,6 @@ spec:
   nodes:
     edpm-compute-0:
       hostName: edpm-compute-0
-  networkAttachments:
-    - ctlplane
-  nodeTemplate:
-    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
     networks:
```

Comment:
Hmm.. we should not move this from nodeTemplate section. Though it works, you don't have to repeat it for every node when there are no `fixedIP`s.